www/include/common/javascript/datepicker/localizedDatepicker.js

```diff
@@ -61,8 +61,9 @@ function initDatepicker(className, altFormat, defaultDate, idName, timestampToSe
             // finding all the alternative field
             var altName = jQuery(this).attr("name");
             if (typeof(altName) != "undefined") {
-                var alternativeField = "input[name=alternativeDate" + altName[0].toUpperCase() + altName.slice(1) + "]";
-                var value = $(this) && $(this).val() ? $(this).val() : defaultDate;
+                var alternativeName = "alternativeDate" + altName[0].toUpperCase() + altName.slice(1);
+                var alternativeField = "input[name=" + alternativeName + "]";
+                var value = $('input[name='+alternativeName+']').val() ? new Date($('input[name='+alternativeName+']').val()) : ($(this) && $(this).val() ? $(this).val() : defaultDate);
```

Comment:
Thanks again @CPbN ,
I'll test ASAP, if the "new Date" correctly compute the localization and timezone of the user

Reply:
Hi @sc979, it does :wink:
Thank you very much for testing :+1: